CedarJavaFFI/src/interface.rs

```diff
@@ -201,6 +176,22 @@ fn parse_schema_internal<'a>(
     }
 }
 
+fn parse_human_schema_internal<'a>(
+    env: &mut JNIEnv<'a>,
+    schema_jstr: JString<'a>,
+) -> Result<JValueOwned<'a>> {
+    if schema_jstr.is_null() {
+        raise_npe(env)
+    } else {
+        let schema_jstring = env.get_string(&schema_jstr)?;
+        let schema_string = String::from(schema_jstring);
+        match Schema::from_str_natural(&schema_string) {
+            Err(e) => Err(Box::new(e)),
+            Ok(_) => Ok(JValueGen::Object(env.new_string("Success")?.into())),
```

Comment:
Returns the string `Success` on success, but on the Java side, `Schema.parse()` seems to expect it to return the schema text?

Reply:
Ah, that is indeed not doing the right thing (and testing didn't catch it since we don't have any tests that do something with a parsed schema). Will fix in next commit.

Reply:
Curious why return the schema contents?  Java already had a string containing the schema contents, I think?

Reply:
Yes, Java already has the string -- the only new information "returned" by the parsing function is whether an exception was thrown or not. Looking at it again, it likely saves some time not to pass back the string, so I'll revert to that.